src/main/java/com/connexta/store/clients/IndexClient.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.clients;
+
+import java.net.URL;
+import java.util.UUID;
+
+public interface IndexClient {
+
+  void indexDataset(UUID datasetId, URL fileUrl, URL irmUrl, URL metacardUrl);
```

Comment:
✏️ Document exceptions

Reply:
I opted to just remove and not document. Since they are runtime exceptions which are translated into HTTP responses, they should never be caught by a caller anyways. Eventually we need to add checked exceptions to this.